### DISCUSSION TITLE
helm chart changing meaning of ingress.enabled was unexpected

### DISCUSSION THREAD

**Discussion by roderickgreen** [SeleniumHQ/docker-selenium#2372] [General]
Just general feedback: changing the meaning of ingress.enabled from "create an ingress resource" to "install an ingress controller" cost me a decent amount of time to track down.  I'd personally reconsider that change because most charts in the wild use ingress.enabled to create an Ingress resource.

**Comment by VietND96** [comment]
Thanks for your feedback! Let me recheck on this

**Reply by VietND96** [reply]
@roderickgreen it is back in chart version 0.35.0

**Reply by roderickgreen** [reply]
Thank you @VietND96!
